src/frontend/src/index.less

```diff
@@ -56,7 +56,17 @@ a {
   }
 
   .-sort-desc:after {
-    transform: translateY(0px) rotateX(-180deg) rotateZ(-135deg);    
+    transform: translateY(0px) rotateX(-180deg) rotateZ(-135deg);
+  }
+}
+
+.error {
+  background-color: @navRod;
+  padding: 1rem;
+  color: white;
+
+  &:empty{
```

Comment:
Egentlig burde vi ikke ha tomme error-bokser om feilen ikke vises. ikke så viktig her. Men i andre applikasjoner som brukes av blinde vil dette gjøre det vanskelig for skjermlesere.

Vet ikke helt om jeg synes det er verd å endre her. men legger inn kommentar så man er klar over det for andre apper :)